resources/profiles/Prusa.json

```diff
@@ -19,6 +19,14 @@
         {
             "name": "MINI",
             "sub_path": "machine/Prusa MINI.json"
+        },
+        {
+            "name": "XL",
+            "sub_path": "machine/Prusa XL.json"
+        },
+        {
+            "name": "XLIS",
```

Comment:
Same, shall we call it "Prusa XL Inut Shaper" here?

Reply:
I think we should do "Prusa XLIS" as that is how is is named in PrusaSlicer. However, if you want it like "Prusa XL Inut Shaper", you got it!

Reply:
lol, must be another issue on my side